Parachain not producing any block: expected 0 to be at least 10

I was following your guide https://paritytech.github.io/zombienet/docs/book/test-dsl-definition-spec.html#available-assertions
I copied this line below from that Available Assertions and pasted it into 0001-small-network.feature file
```alice: parachain 100 block height is at least 10 within 250 seconds```

Then I ran the test:
```./zombienet-linux -p kubernetes test zombienet/0001-small-network.feature```

Then the output:
```
    ✔ alice: is up (162ms)
    ✔ bob: is up (93ms)
2022-05-04 16:31:17        API/INIT: RPC methods not decorated: beefy_getFinalizedHead, state_trieMigrationStatus
    ✔ alice: parachain 100 is registered within 225 seconds (3026ms)

	 Error:  
		 createType(Header):: decodeU8a: failed at 0xf10d89ba1e7bce… on extrinsicsRoot: H256:: Expected at least 32 bytes (256 bits), found 7 bytes

    1) alice: parachain 100 block height is at least 10 within 250 seconds
    ✔ alice: reports node_roles is 4
    ✔ alice: reports sub_libp2p_is_major_syncing is 0
    ✔ alice: reports histogram polkadot_pvf_execution_time has at least 2 samples in buckets ["0.1", "0.25", "0.5", "+Inf"] within 100 seconds (57673ms)
    ✔ bob: log line matches glob "*rted #1*" within 10 seconds (93ms)
    ✔ bob: log line matches "Imported #[0-9]+" within 10 seconds (5387ms)
2022-05-04 16:32:24        API/INIT: RPC methods not decorated: beefy_getFinalizedHead, state_trieMigrationStatus
    ✔ bob: system event contains "A candidate was included" within 20 seconds (5463ms)
2022-05-04 16:32:30        API/INIT: RPC methods not decorated: beefy_getFinalizedHead, state_trieMigrationStatus
    ✔ alice: system event matches glob "*was backed*" within 10 seconds (6004ms)

	 Node's logs are available in /tmp/zombie-565dcdb9f5d009de77be982a45df2e00_-184760-mHpt5Q0PMcI9/logs

	 Deleting network
child process exited


  10 passing (4m)
  1 failing

  1) small network( Small Network test )
       alice: parachain 100 block height is at least 10 within 250 seconds:

      AssertionError: expected 0 to be at least 10
      + expected - actual

      -0
      +10
      
      at /snapshot/zombienet/dist/test-runner/index.js:265:37
      at Generator.next (<anonymous>)
      at fulfilled (/snapshot/zombienet/dist/test-runner/index.js:24:58)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

It seems the parachain is not producing any block!???
What is this error below?
```
createType(Header):: decodeU8a: failed at 0xf10d89ba1e7bce… on extrinsicsRoot: H256:: Expected at least 32 bytes (256 bits), found 7 bytes
```
How can I solve this? What is wrong? please advise. Thank you

Hi @pepoviola , thank you for your quick reply!
Yes, that solves that problem... but now there is a new problem...
This test below is failing now. I tried to increase it from 10 to 20 seconds but it still fails due to timeout:
```bob: log line matches glob "*rted #1*" within 20 seconds```
Maybe this error is triggered by the [types.Header] ???

```
	 Error: Timeout(20), "getting log pattern *rted #1* within 20 secs".
    1) bob: log line matches glob "*rted #1*" within 20 seconds

  10 passing (6m)
  1 failing
  1) small network( Small Network test )
       bob: log line matches glob "*rted #1*" within 20 seconds:
     AssertionError: expected false to be truthy
      at /snapshot/zombienet/dist/test-runner/index.js:347:32
      at Generator.next (<anonymous>)
      at fulfilled (/snapshot/zombienet/dist/test-runner/index.js:24:58)
```

--- a/src/networkNode.ts
+++ b/src/networkNode.ts
@@ -313,11 +313,10 @@ export class NetworkNode implements NetworkNodeInterface {
     try {
       const re = isGlob ? minimatch.makeRe(pattern) : new RegExp(pattern, "ig");
       const client = getClient();
-
+      let logs = await client.getNodeLogs(this.name, undefined,  true);
       const getValue = async () => {
         let done = false;
         while (!done) {
-          const logs = await client.getNodeLogs(this.name, 2, true);
           const dedupedLogs = this._dedupLogs(
             logs.split("\n"),
             client.providerName === "native"
@@ -337,6 +336,7 @@ export class NetworkNode implements NetworkNodeInterface {
             debug(this.lastLogLineCheckedTimestamp.split(" ").slice(1).join(" "));
           } else {
             await new Promise((resolve) => setTimeout(resolve, 1000));
+            logs = await client.getNodeLogs(this.name, 2,  true);
           }
         }
       }

